{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2017-15133",
    "ASSIGNER": "secalert@redhat.com"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "CWE-400"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://github.com/miekg/dns/issues/627",
        "name": "https://github.com/miekg/dns/issues/627",
        "refsource": "CONFIRM",
        "tags": [
          "Issue Tracking",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1538763",
        "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1538763",
        "refsource": "CONFIRM",
        "tags": [
          "Issue Tracking",
          "Third Party Advisory"
        ]
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "A denial of service flaw was found in miekg-dns before 1.0.4. A remote attacker could use carefully timed TCP packets to block the DNS server from accepting new connections."
      }
    ]
  }
}